packages/bot/src/stores/flyout-store.js

```diff
@@ -6,6 +6,7 @@ export default class FlyoutStore {
     block_listeners = [];
     block_workspaces = [];
     flyout_min_width = 400;
+    startScale = 0.7;
```

Comment:
is it make more sense if we put this `startScale` at `const.js`?

Reply:
as discussed, we leave it as is for now as we might want to add startScale as parameter in future for the initBlockWorkspace method